docs/Windows-README.md

```diff
@@ -18,47 +18,51 @@ available from Microsoft.com (search for "visual c 2008 redistributable").
 Alternatively, you can install the redistributable in a "side-by-side"
 configuration, which does not require administrator privileges. Simply
 download the DLL package from:
-https://download.savannah.gnu.org/releases/rdiff-backup/Microsoft.VC90.zip
+[https://download.savannah.gnu.org/releases/rdiff-backup/Microsoft.VC90.zip](https://download.savannah.gnu.org/releases/rdiff-backup/Microsoft.VC90.zip)
 and copy the four enclosed files to the same directory as rdiff-backup.exe.
 
 You will need to follow either method only once.
 
 
 ADDITIONAL ISSUES:
 
-Currently, rdiff-backup's --include and --exclude options do not support
-Windows paths with \ as the directory separator. Instead, it is necessary to
+Currently, rdiff-backup's `--include` and `--exclude` options do not support
+Windows paths with \\ as the directory separator. Instead, it is necessary to
 use / which is the Unix directory separator.
 
 Additionally, you may need to run rdiff-backup from the same directory as the
 source of your backup, eg:
 
-> cd c:\
-> rdiff-backup.exe --include "c:/My Stuff" --exclude "c:/**" c:/ c:/Backup
+`> cd c:\`
```

Comment:
That's not the proper way to do code paragraphs, it must be three backticks before or after the paragraph *or* 4 blanks at the beginning of each line. Remove the greater than sign in all cases (and the single backticks, they're only meant for code snippets within a normal paragraph).

Reply:
You are right but that is not what the greater than sign is for.  It was in your original doc and is just indicating that it is a DOS prompt I think.
Anyway, I've reverted to 4 blanks, as the three backticks is "git extension" so not necessarily on all, but probably is in most.

Reply:
It actually helps if I actually read the document I'm updating, rather than just skimming through it.
The little job took longer than the much bigger changes.